website/docs/language/settings/backends/azurerm.mdx

```diff
@@ -295,11 +564,11 @@ When authenticating using the Storage Account's Access Key - the following field
 
 ***
 
-When authenticating using AzureAD Authentication - the following fields are also supported:
+When authenticating using Entra ID service principal, you have the option to use Entra ID authentication for the storage account (as opposed to Access Key or SAS Token) - the following fields are also supported:
 
-* `use_azuread_auth` - (Optional) Should AzureAD Authentication be used to access the Blob Storage Account. This can also be sourced from the `ARM_USE_AZUREAD` environment variable.
+* `use_azuread_auth` - (Optional) Should EntraID Authentication be used to access the Blob Storage Account. This can also be sourced from the `ARM_USE_AZUREAD` environment variable.
```

Comment:
```suggestion
* `use_azuread_auth` - (Optional) Whether Azure Active Directory Authentication should be used to access the Blob Storage Account. This can also be sourced from the `ARM_USE_AZUREAD` environment variable.
```